web/history.md

```diff
@@ -0,0 +1,391 @@
+# KeymanWeb Version History
+
+## 10.0 alpha
+* Updated versioning scheme for uniformity across all Keyman products.
+* Reworked the KeymanWeb attachment model significantly to allow greater user control. (#114)
```

Comment:
From the [history.md](https://github.com/keymanapp/keyman/wiki/history.MD-format) format
> * release note (#issue)

For citations, we want (#issue) instead of (#pr).  I'm assuming this applies to versions 10.0 alpha down to 2.0.453 stable

Reply:
I thought that might have been it.  This being stated, you'll note that some of the old entries on that page used (#pr) instead, so that got me confused.

Reply:
Not all of the #prs listed there have corresponding issues, so I've left those numbers in place.  The others have been changed into their corresponding issue.